BDArmory/Control/BDModuleOrbitalAI.cs

```diff
@@ -1802,6 +1802,8 @@ private void UpdateEngineLists(bool forceUpdate = false)
                 {
                     reverseEngines.Add(engine);
                     reverseThrust += engine.MaxThrustOutputVac(true);
+                    var gimbal = engine.part.FindModuleImplementing<ModuleGimbal>();
+                    if (gimbal != null) gimbal.gimbalLock = true; // Disable gimbal on reverse/RCS engines since they don't work for directions other than forward
```

Comment:
Would it be worth setting `gimbal.gimbalRange = 0` instead to avoid accidentally locking them in a non-neutral position or is that unlikely enough not to worry about?